pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/json/UserActivity.java

```diff
@@ -2,41 +2,25 @@
 
 import com.google.gson.annotations.SerializedName;
 
+import lombok.AllArgsConstructor;
+import lombok.Value;
 import org.broadinstitute.ddp.json.activity.TranslatedSummary;
 
+@Value
+@AllArgsConstructor
 public class UserActivity implements TranslatedSummary {
-
     @SerializedName("guid")
-    private String guid;
+    String guid;
     @SerializedName("title")
-    private String title;
+    String title;
     @SerializedName("subtitle")
-    private String subtitle;
+    String subtitle;
     @SerializedName("typeName")
-    private String typeName;
+    String typeName;
     @SerializedName("status")
-    private String status;
+    String status;
     @SerializedName("isoLanguageCode")
-    private String isoLanguageCode;
-
-    /**
-     * Instantiate UserActivity object.
-     */
-    public UserActivity(
-            String guid,
-            String title,
-            String subtitle,
-            String typeName,
-            String status,
-            String isoLanguageCode
-    ) {
-        this.guid = guid;
-        this.title = title;
-        this.subtitle = subtitle;
-        this.typeName = typeName;
-        this.status = status;
-        this.isoLanguageCode = isoLanguageCode;
-    }
+    String isoLanguageCode;
```

Comment:
Can we add a `@Getter(onMethod = @__(@Override))` to this field and remove the manual getter implementation?